CrewChiefV4/speech_recognition_config.txt

```diff
@@ -218,3 +226,12 @@ POSSESSIVE = 's
 48 = fourty-eight
 49 = fourty-nine
 50 = fifty
+51 = fifty-one
+52 = fifty-two
+53 = fifty-three
+54 = fifty-four
+55 = fifty-five
+56 = fifty-six
+57 = fifty-seven
+58 = fifty-eight
+59 = fifty-nine
```

Comment:
add 60 = sixty

because some folks will run 1 hour races and expect the app to understand "60 minutes". I wouldn't bother with anything over 60